code/frameworks/angular/src/types.ts

```diff
@@ -46,4 +46,5 @@ export type StorybookConfig = Omit<
 
 export interface AngularOptions {
   enableIvy: boolean;
+  enableNgcc: boolean;
```

Comment:
This (and the line above that is not coming from your PR) should probably be 

```suggestion
  enableNgcc?: boolean;
```
as it is not guaranteed that it exists in the configuration of an app

What do you think?

Reply:
I also made `enableIvy` optional. As by default it is true.